models/owners.model.js

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-console */
+import bcrypt from 'bcrypt';
+import shortid from 'shortid';
+import mongoose from 'mongoose';
+
+import Owner from '../schemas/owners.schema';
```

Comment:
You're not importing a schema here, but a model (so you're importing a model inside a model). It's normal that you try to separate these two things, but it not needed here.

You can create the Mongoose model here and then append some class methods (`createOwner`) into it. 

The result is good anyways, as you are protecting Mongoose logic from the controller, separating correctly the controller layer from the database layer.